apps/web/components/ApiCatalogueFilter/ApiCatalogueFilter.tsx

```diff
@@ -1,96 +0,0 @@
-import {
```

Comment:
Why delete this file, it's a component being used here: apps/web/screens/Organization/StafraentIsland/ApiCatalogue.tsx